### ISSUE TITLE
Unable to generate reports with empty enhanced table [Kibana 7.5.0 - ET 1.6.0-1.8.0]

### ISSUE THREAD

**Issue by fabiopipitone** [fbaligand/kibana-enhanced-table#93]
Hi there, 

I'd like to highlight a possible bug which can cause quite some problems in a production environment. As described by the title, I've experienced a fail in the report when the enhanced tables happen to be empty (`No results found`). It doesn't happen with default tables. 

Also, I want to specify I'm in a air-gapped environment (in fact, due to a Kibana issue in such an environment when it comes to make report, I had to specify `newsfeed.enabled: false` in the kibana.yml).

Steps to reproduce: 

- download brand-new tars of elasticsearch and kibana version 7.5.0, unzip it and launch elasticsearch.
- install enhanced table on the kibana instance and launch kibana.
- go to `Management>Elasticsearch>License Management` and activate the trial license (to unlock reporting features).
- put a template on a given `testing-index` (needed only to ingest a document with a recognized date) and then ingest a document. You can do both from the Dev Tools this way
```
PUT _template/testing_template
{
  "order": 2,
  "index_patterns": [
    "testing*"
  ],
  "settings": {
    "index": {
      "number_of_shards": "1",
      "number_of_replicas": "0"
    }
  },
  "mappings": {
    "properties": {
      "date": {
        "type": "date"
      }
    }
  },
  "aliases": {}
}

PUT testing_index/_doc/1
{
  "foo": "bar",
  "date": "2020-03-11T06:00:00.000Z"
}
```
- Make a index-pattern `testing_index*`.
- Go to the Dashboard section, create a new Dashboard (e.g. `testing dashboard`), click on `Add` to add a visualization and create a enhanced table one like the following:

![image](https://user-images.githubusercontent.com/45557452/76397402-49e39400-637b-11ea-845b-9f856e849227.png)

- Once the visualization is added to the dashboard, set a timerange in the dashboard to visualize something in the enhanced table (so include the `date` of the document in your timerange). 
- Save the dashboard storing the time with it. 
- Try making a report from inside the Dashboard (once it is saved) by clicking on `Share>PDF Reports>Generate PDV`. You should see in the bottom right corner of the screen a green pop-up like the following that indicates the report has been queued: 
![image](https://user-images.githubusercontent.com/45557452/76397782-fc1b5b80-637b-11ea-85c7-e158ec8435eb.png)

- After few moments you should be able to download the report from the `Management>Kibana>Reporting` section.

- Now, try to edit your Dashboard, setting a timerange which DOES NOT include your document. You should see something like this:

![image](https://user-images.githubusercontent.com/45557452/76397894-34229e80-637c-11ea-947d-bba6f15e713f.png)

- Save the Dashboard (storing time again) and try to make another report. 

In my case, the report fails because it reaches the maximum number of retries. 
As already said, it doesn't happen with default tables (you can try following the above steps but with a classic Table visualization and it'll work). 

Hope this can be sorted out soon since the reporting feature is kinda useful and utilised in prod environments and it can happen there are no data for a specific timerange. It would prevent someone from using this super useful visualization. 

Thanks.

**Comment by fbaligand** [comment]
Hi, 
I managed to reproduce your issue, and so, I confirm that there is a bug.
That said, I'm annoyed because I have no precise error, no log that can help me to understand why this issue happens...

**Comment by fabiopipitone** [comment]
Yeah, I know. That annoys me too. 

Might it be linked to any kind of timeout that causes the report to fail? Maybe because it tries to load something for an undefined time, breaking the report. 

I don't know really, but it prevents anybody from using this visualization anywhere where a report is needed.

**Comment by fbaligand** [comment]
Well, I'm as annoyed as you.
I would like to fix the issue.
But I have no idea what is the cause of this issue.

**Comment by fbaligand** [comment]
Hi, 
I just published Enhanced Table v1.11.0! And it includes a fix of this bug.

More info on release here:
https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.11.0
